cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommand.java

```diff
@@ -207,7 +209,7 @@ private void sendFailureNotifications(List<CliCheckResult> failures, boolean har
     private void checkForHardFail(List<CliCheckResult> results) {
         if (getCheckerHardFailures(results).count() > 0) {
             String cliFailureString = "The following checks had hard failures:" + System.lineSeparator() +
-                    getCheckerHardFailures(results).map(failure -> failure.getCheckName() + " failures: " + System.lineSeparator() + failure.getNotificationText() + System.lineSeparator())
+                    getCheckerHardFailures(results).map(failure -> failure.getCheckName() + " failures: " + System.lineSeparator() + failure.getNotificationText().replaceAll(QUOTE_MARKER, "'") + System.lineSeparator())
```

Comment:
keeping back-tick might be more readable actually. At least in Phabricator, I found the quote misleading when the English string had one, which prompted to ask for using the code block. 

Using function too?